src/services/consent/services/consent.deprecated.js

```diff
@@ -0,0 +1,125 @@
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
```

Comment:
Codacy found an issue: ['Forbidden' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5680233)